mysql/errcode.go

```diff
@@ -893,6 +893,7 @@ const (
 
 	// TiDB self-defined errors.
 	ErrMemExceedThreshold = 8001
+	ErrForUpdateCantRetry = 8002
```

Comment:
Do not forget to update the document. https://github.com/pingcap/docs-cn/blob/master/sql/error.md